### ISSUE TITLE
Variable name parsing issue

### ISSUE THREAD

**Issue by DanPurdy** [tonyganch/gonzales-pe#226]
In the following example

``` scss
.test {
  padding: $mobile-site-gutter*1.5;
}
```

The ident would be parsed as `$mobile-site-gutter*1`

``` json
 -> stylesheet
 | -> ruleset
 | | -> selector
 | | | -> class
 | | | | -> ident
 | | | |    "test"
 | | -> space
 | |    " "
 | | -> block
 | | | -> space
 | | |    "\n  "
 | | | -> declaration
 | | | | -> property
 | | | | | -> ident
 | | | | |    "padding"
 | | | | -> propertyDelimiter
 | | | |    ":"
 | | | | -> space
 | | | |    " "
 | | | | -> value
 | | | | | -> variable
 | | | | | | -> ident
 | | | | | |    "mobile-site-gutter*1"
 | | | | | -> number
 | | | | |    ".5"
 | | | -> declarationDelimiter
 | | |    ";"
 | | | -> space
 | | |    "\n"
```

It's fine with CSS

as reported in https://github.com/sasstools/sass-lint/issues/901

**Comment by tonyganch** [comment]
I'll take a look at this one.

CSS is not okay, at least it looks strange to have two indents and number:

```
> echo 'nani*1' | ./bin/gonzales.js --syntax css --context value --simple -                                                                                             

 -> value
 | -> ident
 |    "nani"
 | -> ident
 |    "*"
 | -> number
 |    "1"
```

**Comment by tonyganch** [comment]
I've released the fix as `v3.4.6` but wasn't able to test `sass-lint` with it locally: cli tests fail for me with fresh install.
Could you please check if everything is okay? I've changed quite a bit there.
